Assets/IntegrationTests/TestShopifyCollections.cs

```diff
@@ -1,45 +1,40 @@
 namespace Shopify.Unity.Tests
 {
     using UnityEngine;
+	using UnityEngine.TestTools;
+	using NUnit.Framework;
     using Shopify.Unity;
+	using System.Collections;
     using System.Collections.Generic;
 
-    [IntegrationTest.DynamicTest ("TestScene")]
-    [IntegrationTest.SucceedWithAssertions]
     public class TestLoad3Collections : MonoBehaviour {
-        void Start() {
+		[UnityTest]
+		public IEnumerator Load3Collections() {
             ShopifyBuy.Init("351c122017d0f2a957d32ae728ad749c", "graphql.myshopify.com");
+			bool hadResult = false;
 
             ShopifyBuy.Client().collections(
-                first: 3,
+                first: 1,
```

Comment:
Should this test now be called "loadCollection" if it only loads one collection? Because the new test stack is unit-testey do you plan on following up by combining these tests into one file?